src/terra-dev-site/doc/how-to.2/consume-terra-application.0.doc.mdx

```diff
@@ -0,0 +1,15 @@
+# How To Consume Terra Application
+
+Due to the large number of Context-based APIs provided by `terra-application`, the recommended strategy for consuming `terra-application` is to define it as a `peerDependency` within your `package.json`.
```

Comment:
> the recommended strategy for consuming `terra-application` is to define it as a `peerDependency` within your `package.json`.

It might be worth mentioning the application package / server level should be the only project that should define it as a hard dependency to reiterate: https://github.com/cerner/terra-application/pull/2/files#diff-dd3b6db2461ac3f1b54c8bd2cd30b6bfR7